batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,45 @@
+package com.lftechnology.batch7crud.filter;
+
+import com.lftechnology.batch7crud.constants.AttributeConstants;
+import com.lftechnology.batch7crud.constants.UrlConstants;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+/**
+ * Created by Romit Amgai <romitamgai@lftechnology.com> on 1/26/16.
+ */
+
+public class AuthenticationFilter implements Filter {
+
+  @Override
+  public void init(FilterConfig filterConfig) throws ServletException {
+    //Initial params can be handled here
+  }
+
+  @Override
+  public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain)
+          throws IOException, ServletException {
+    //filterChain.doFilter(servletRequest, servletResponse);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)